tests/production/headers/test_cors.py

```diff
@@ -1,8 +1,7 @@
 import requests
 import pytest
 from lib import Assertions, Authentication
-from lib.constants.constants import PROD_URL, VALID_ENDPOINTS
-
+from lib.constants.constants import PROD_URL, VALID_ENDPOINTS, ORIGIN
 
 METHODS = ["GET", "POST", "PUT", "PATCH", "DELETE"]
 TEST_METHODS = ["get", "post", "put", "delete"]

```

Comment:
hmmm, why do we have these two lists of methods?

Reply:
As above, there existence is unclear